docs/best-practices.md

```diff
@@ -60,6 +60,12 @@ There's no silver bullet for this problem, and the solution strictly depends on
 - The [init configuration](https://falco.org/docs/configuration/#plugins) should contain information that is used during the whole plugin lifecycle and that is used across both field extraction and event generation
 - The init configuration is the right place for structured data. In fact, in most cases, plugins accept JSON strings as a configuration and also expose a schema describing/documenting the expected data format (see [`plugin_get_init_schema`](https://falco.org/docs/plugins/plugin-api-reference/#const-char-plugin-get-init-schema-ss-plugin-schema-type-schema-type-required-no) for more details)
 - The open parameters should contain information that is only relevant for opening a specific event source, and their lifecycle ends at the invocation of `plugin_close()`
+- Open parameters should have the following annotations:
```

Comment:
```suggestion
- Init configuration parameters should have the following annotations:
```
Annotations can only be used in init configuration if an init schema is defined. So, I'd also move this under the point that talks about the `plugin_get_init_schema` (line 61)

Reply:
May it be worth also adding a link to the official specification :point_down: ?

https://json-schema.org/draft/2020-12/json-schema-validation.html#name-a-vocabulary-for-basic-meta

Reply:
Done.